### PR TITLE
added in es adapters

### PR DESCRIPTION
**Pull request by SanaaYousaf**

### CODE REVIEW

**Review by giorod3 — DISMISSED** [review]

**Comment by giorod3** [comment]
@simar7 please also review and merge if you feel it looks good

**Comment by simar7** [comment]
looks like some tests are failing, should be easy to fix.

File: internal/adapters/terraform/aws/elasticsearch/adapt.go [hunk 1]
```diff
@@ -24,11 +24,15 @@ func adaptDomains(modules terraform.Modules) []elasticsearch.Domain {
 
 func adaptDomain(resource *terraform.Block) elasticsearch.Domain {
 	domain := elasticsearch.Domain{
-		Metadata:   resource.GetMetadata(),
-		DomainName: defsecTypes.StringDefault("", resource.GetMetadata()),
+		Metadata:               resource.GetMetadata(),
+		DomainName:             defsecTypes.StringDefault("", resource.GetMetadata()),
+		AccessPolicies:         resource.GetAttribute("access_policies").AsStringValueOrDefault("", resource),
+		VpcId:                  resource.GetAttribute("vpc_options.0.vpc_id").AsStringValueOrDefault("", resource),
```
Comment (simar7) [inline internal/adapters/terraform/aws/elasticsearch/adapt.go:30]: how did you get these attributes? What if there's more than one VPC option? I assume `vpc_options.0` only supports the first?

File: internal/adapters/cloudformation/aws/elasticsearch/domain.go [hunk 2]
```diff
@@ -26,18 +30,27 @@ func getDomains(ctx parser.FileContext) (domains []elasticsearch.Domain) {
...
+				UpdateAvailable: defsecTypes.Bool(false, r.Metadata()),
+			},
 		}
 
 		if prop := r.GetProperty("LogPublishingOptions"); prop.IsNotNil() {
 			domain.LogPublishing = elasticsearch.LogPublishing{
-				Metadata:     prop.Metadata(),
-				AuditEnabled: prop.GetBoolProperty("AUDIT_LOGS.Enabled", false),
+				Metadata:              prop.Metadata(),
+				AuditEnabled:          prop.GetBoolProperty("AUDIT_LOGS.Enabled", false),
```
Comment (simar7) [inline internal/adapters/cloudformation/aws/elasticsearch/domain.go:52]: ditto here. is there a resource I can read which explains these fields?

File: internal/adapters/terraform/aws/elasticsearch/adapt.go (hunk 1 above)
Comment (SanaaYousaf) [reply internal/adapters/terraform/aws/elasticsearch/adapt.go:30]: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elasticsearch_domain...I follow this documentation

File: internal/adapters/cloudformation/aws/elasticsearch/domain.go (hunk 2 above)
Comment (SanaaYousaf) [reply internal/adapters/cloudformation/aws/elasticsearch/domain.go:52]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticsearch-domain.html

File: internal/adapters/terraform/aws/elasticsearch/adapt.go (hunk 1 above)
Comment (simar7) [reply internal/adapters/terraform/aws/elasticsearch/adapt.go:30]: perfect

File: internal/adapters/cloudformation/aws/elasticsearch/domain.go (hunk 2 above)
Comment (simar7) [reply internal/adapters/cloudformation/aws/elasticsearch/domain.go:52]: thanks makes sense

**Review by simar7 — APPROVED** [review]

**Pull request merged**